preview_generator/preview/builder/image__imconvert.py

```diff
@@ -83,14 +83,16 @@ def build_jpeg_preview(
         preview_name: str,
         cache_path: str,
         page_id: int,
-        extension: str = ".jpg",
-        size: ImgDims = None,
+        extension: str,
+        size: ImgDims,
         mimetype: str = "",
     ) -> None:
         # inkscape tesselation-P3.svg  -e
-        tempfolder = tempfile.tempdir
         tmp_filename = "{}.png".format(str(uuid.uuid4()))
-        tmp_filepath = os.path.join(tempfolder, tmp_filename)
```

Comment:
This one was failing if `tempfile.tempdir` was `None`, it can, according to the doc.